Fitbit.Portable/FitBitHttpClientMessageHandler.cs

```diff
@@ -1,38 +1,59 @@
 ﻿namespace Fitbit.Api.Portable
 {
+    using System;
     using System.Diagnostics;
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
 
-    public class FitbitHttpClientMessageHandler : DelegatingHandler
```

Comment:
Interesting.... in my local repo the file of the name is using the correct lowercase 'b' and it shows it is not modified. Can you try to clone locally and see what you get?